instrumentation/go.mongodb.org/mongo-driver/v2/mongo/otelmongo/version.go

```diff
@@ -0,0 +1,10 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package otelmongo // import "go.opentelemetry.io/contrib/instrumentation/go.mongodb.org/mongo-driver/v2/mongo/otelmongo"
+
+// Version is the current release version of the mongo-go-driver V2 instrumentation.
+func Version() string {
+	return "0.60.0"
```

Comment:
cc @MrAlias 

This kinds of goes in the same way as this comment: https://github.com/open-telemetry/opentelemetry-go-contrib/pull/6961#discussion_r2003056452

We should therefore adopt the same behavior. So let's keep commit hashes rather than releases for now (you just need to remove the entries in `versions.yaml`).

Reply:
Sounds good.  I'll remove the entries from versions.yaml.  I'm unsure what needs to be done about the commit hashes, as I don't see any action taken yet with the referenced PR.

Reply:
You should still add it to excluded modules in `versions.yaml`, so it doesn't get released. But it gets dependency upgrades on a new release.